pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -451,9 +451,11 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
 
   protected async runWslProxy() {
     const debug = this.debug ? 'true' : 'false';
+    const logDir = await this.wslify(paths.logs);
+    const logfile = path.join(logDir, 'wsl-proxy.log');
```

Comment:
```suggestion
    const logDir = await this.wslify(path.join(logDir, 'wsl-proxy.log');
```
[`path.join`](https://nodejs.org/docs/latest-v22.x/api/path.html#pathjoinpaths) uses the platform-native path separator, so `\` on Windows. That gets passed to a Linux program, so that's the incorrect separator (and might get treated as an escape character instead).